webapp/portlet/src/main/webapp/common/components/ExoIdentitySuggester.vue

```diff
@@ -15,7 +15,8 @@
       :height="height"
       :filter="filterIgnoredItems"
       :hide-no-data="hideNoData"
-      :class="(required && !value && 'required-field invalid') || (required && 'required-field')"
+      :class="(required && !value && 'required-field invalid') || (required && 'required-field') || (innerIcon && 'customIdentitySuggester' && 'customIconIdentitySuggester') || (!innerIcon && 'identitySuggesterStyle')"
```

Comment:
Add a computed property to : 
1/ use required style or not
2/ use `identitySuggesterFilterStyle` class

```vue.js
computed: {
  autocompleteClass() {
    const requiredClass = (this.required && !this.value && 'required-field invalid') || (this.required && 'required-field') || '';
    const sugesterStyleClass = this.filterStyle && 'identitySuggesterFilterStyle' || 'identitySuggesterInputStyle';
    return `${requiredClass} ${sugesterStyleClass}`;
  }
}
```